src/Raven.Server/Web/Studio/Sharding/Processors/ShardedBucketsHandlerProcessorForGetBuckets.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Executors;
+using Raven.Server.Documents.Sharding.Handlers;
+using Raven.Server.Documents.Sharding.Operations;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Web.Studio.Processors;
+
+namespace Raven.Server.Web.Studio.Sharding.Processors
+{
+    internal class ShardedBucketsHandlerProcessorForGetBuckets : AbstractBucketsHandlerProcessorForGetBuckets<ShardedDatabaseRequestHandler, TransactionOperationContext>
+    {
+        public ShardedBucketsHandlerProcessorForGetBuckets([NotNull] ShardedDatabaseRequestHandler requestHandler) : base(requestHandler)
+        {
+        }
+        
+        protected override async ValueTask<BucketsResults> GetBucketsResults(TransactionOperationContext context, int fromBucket, int toBucket, int range, CancellationToken token)
+        {
```

Comment:
you should be able to request the info for a specific shard here as well

Reply:
done

Reply:
do we need to point to a specific nodeTag as well?